path_config.go

```diff
@@ -97,14 +98,26 @@ func (b *kubeAuthBackend) pathConfigWrite(ctx context.Context, req *logical.Requ
 		return logical.ErrorResponse("no host provided"), nil
 	}
 
+	localCACert, _ := ioutil.ReadFile("/var/run/secrets/kubernetes.io/serviceaccount/ca.crt")
+
+	localTokenReviewer, _ := ioutil.ReadFile("/var/run/secrets/kubernetes.io/serviceaccount/token")
```

Comment:
These errors should not be discarded since if they don't exist, it will cause problems.

Reply:
If the errors are nil, the byte slice will also be nil, so will be converted into an empty string later. It’s only if both the default files and the configured cert data are empty that we have an error?

Reply:
That's fair!